specs/_features/ePBS/validator.md

```diff
@@ -0,0 +1,243 @@
+
+All terminology, constants, functions, and protocol mechanics defined in the updated Beacon Chain doc of ePBS. are requisite for this document and used throughout.
+Please see related Beacon Chain doc before continuing and use them as a reference throughout.
+
+## Protocols
+
+### `ExecutionEngine`
+
+*Note*: `get_inclusion_list` function is added to the `ExecutionEngine` protocol for use as a validator.
+
+The body of this function is implementation dependent.
+The Engine API may be used to implement it with an external execution engine.
+
+#### `get_inclusion_list`
+
+Given the `payload_id`, `get_payload` returns `GetInclusionListResponse` with the most recent version of
+the execution payload that has been built since the corresponding call to `notify_forkchoice_updated` method.
+
+```python
+def get_inclusion_list(self: ExecutionEngine, payload_id: PayloadId) -> GetInclusionListResponse:
+    """
+    Return ``GetInclusionListResponse`` object.
+    """
+    ...
+```
+
+## Beacon chain responsibilities
+
+All validator responsibilities remain unchanged other than those noted below. Namely, proposer normal block production switched to a new `BeaconBlockBody`.
+Proposer with additional duty to construct and broadcast `InclusionList` alongside `SignedBeaconBlock`.
+Attester by chance with an additional duty to be part of PTC committee and broadcast `PayloadAttestationMessage`.
+
+## Validator assignment
+
+A validator can get PTC committee assignments for a given epoch using the following helper via `get_ptc_committee_assignment(state, epoch, validator_index)` where `epoch <= next_epoch`.
+
+A validator can use the following function to see if they are supposed to submit payload attestation message during a slot. 
+This function can only be run with a `state` of the slot in question. PTC committee selection is only stable within the context of the current and next epoch.
+
+```python
+def get_ptc_committee_assignment(state: BeaconState,
+                             epoch: Epoch,
+                             validator_index: ValidatorIndex
+                             ) -> Optional[Tuple[Sequence[ValidatorIndex], Slot]]:
+    """
+    Return the ptc committee assignment in the ``epoch`` for ``validator_index``.
+    ``assignment`` returned is a tuple of the following form:
+        * ``assignment[0]`` is the list of validators in the committee
+        * ``assignment[1]`` is the slot at which the committee is assigned
+    Return None if no assignment.
+    """
+    next_epoch = Epoch(get_current_epoch(state) + 1)
+    assert epoch <= next_epoch
+
+    start_slot = compute_start_slot_at_epoch(epoch)
+    committee_count_per_slot = get_committee_count_per_slot(state, epoch)
+    for slot in range(start_slot, start_slot + SLOTS_PER_EPOCH):
+        for index in range(committee_count_per_slot):
+            committee = get_ptc_committee(state, Slot(slot))
```

Comment:
why aren't you using `get_ptc`  from the beacon-chain spec? https://github.com/potuz/consensus-specs/blob/epbs/specs/_features/epbs/beacon-chain.md#get_ptc